### PR TITLE
Fix 214 sales account

### LINKED ISSUE #214: Fix Customer SalesAccount type
Apparently, `SalesAccount` attribute on `Customer` is a string, not an integer as the `Number` attribute in `Account` is.

According to the Fortnox documentation, in `Customer` a `Sales Account` is a "string 4 characters", but in `Account` the `Numer` attribute is an "integer <int32> [ 1000 .. 9999 ]". So Fortnox has different data types for "the same thing" on different endpoints :weary: 

We are currently using the same implementation for validating both these attributes, but that's not a correct implementation. We get this error:

```
[Fortnox::API::Model::Customer.new] "3010" (String) has invalid type for :sales_account violates constraints
(type?(Integer, "3010") AND gteq?(0, "3010") AND lteq?(9999, "3010") failed) (Fortnox::API::AttributeError)
```

This is how the attribute is defined now, which is obviously wrong:
```
# SalesAccount  Sales account of the customer, 4 digits
attribute :sales_account, Types::AccountNumber
```

### PR DESCRIPTION
**Pull request by ehannes**
Fixes #214. See the issue for more information. The Fortnox API is just stupid here...

### CODE REVIEW

File: lib/fortnox/api/models/customer.rb [hunk 1]
```diff
@@ -161,7 +161,7 @@ class Customer < Fortnox::API::Model::Base
         attribute :project, Types::Nullable::String
 
         # SalesAccount  Sales account of the customer, 4 digits
-        attribute :sales_account, Types::AccountNumber
+        attribute :sales_account, Types::SalesAccount
```
Comment (d-Pixie) [inline lib/fortnox/api/models/customer.rb:164]: Comment needs updating

**Review by d-Pixie — APPROVED** [review]

File: lib/fortnox/api/models/customer.rb (hunk 1 above)
Comment (ehannes) [reply lib/fortnox/api/models/customer.rb:164]: We should probably just get rid of these comments and refer to the documentation instead... These comments will just get old and I don't see the point in keeping them updated. Do you @d-Pixie?

Comment (d-Pixie) [reply lib/fortnox/api/models/customer.rb:164]: No, I agree. I almost wrote " ... or removed" yesterday 😄

Comment (ehannes) [reply lib/fortnox/api/models/customer.rb:164]: I'll change the comment for this PR so we leave this in a correct state.

**Pull request merged**